gtfs/spec/en/reference.md

```diff
@@ -32,6 +33,22 @@ This section defines terms that are used throughout this document.
 * **Field optional** - The field column may be omitted from your feed. If you choose to include an optional column, each record in your feed must have a value for that column. You may include an empty string as a value for records that do not have values for the column. Some optional fields permit an empty string as a value. To enter an empty string, just omit any text between the commas for that field. Note that 0 is interpreted as "a string of value 0", and is not an empty string.
 * **Dataset unique** - The field contains a value that maps to a single distinct entity within the column. For example, if a route is assigned the ID **1A**, then no other route may use that route ID. However, you may assign the ID **1A** to a location because locations are a different type of entity than routes.
 
+## Field Types
+
+- **Ad-hoc** - Please see the detail column of the field for the definition of the field type.
+- **Color** - The field contains a color encoded as a six-character hexadecimal number. Refer to https://htmlcolorcodes.com/ to generate a valid value. E.g. "FFFFFF" for white, "000000" for black or "0039A6" for the A,C,E lines in NYMTA.
```

Comment:
I'm adding the # comment after the "valid value". Because when you read the definition as "a six-character hexadecimal number", you don't see why we speak about not including the #. There is no hash in a six-character hexadecimal number.